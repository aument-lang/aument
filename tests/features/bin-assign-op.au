let x = 0;
x = 1; x += 2;
print x;
x = 2; x -= 2;
print x;
x = 1; x *= 2;
print x;
x = 10; x /= 5;
print x;
x = 11; x %= 2;
print x;