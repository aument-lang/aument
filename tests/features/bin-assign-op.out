int;3
int;0
int;2
float;2.0
int;1